src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/PackageSearchArgs.cs

```diff
@@ -40,7 +44,34 @@ public int VerifyInt(string number, int defaultValue)
                 return verifiedNumber;
             }
 
-            throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Strings.Error_invalid_number, number));
+            throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Strings.Error_invalidOptionValue, number, option));
+        }
+
+        private bool IsJsonFormat(string format)
+        {
+            if (!string.IsNullOrEmpty(format) && string.Equals(format, "json", StringComparison.CurrentCultureIgnoreCase))
+            {
+                return true;
+            }
+
+            return false;
+        }
+
+        private PackageSearchVerbosity GetVerbosityFromOption(string verbosity)
+        {
+            if (verbosity != null)
+            {
+                if (string.Equals(verbosity, nameof(PackageSearchVerbosity.Detailed), StringComparison.CurrentCultureIgnoreCase))
```

Comment:
same here, would it be easier to use an enum?

There's probably examples with other options: https://github.com/NuGet/NuGet.Client/blob/58f5aad3fc1684c20798ab28200aa62fe0512df9/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommand.cs#L174-L182

Reply:
fixed